doc/src/Build_development.rst

```diff
@@ -234,12 +241,14 @@ will be skipped if prerequisite features are not available in LAMMPS.
    time.  Preference is given to parts of the code base that are easy to
    test or commonly used.
 
-Tests for styles of the same kind of style (e.g. pair styles or bond
-styles) are performed with the same test executable using different
-input files in YAML format.  So to add a test for another style of the
-same kind it may be sufficient to add a suitable YAML file.
-:doc:`Detailed instructions for adding tests <Developer_unittest>` are
-provided in the Programmer Guide part of the manual.
+Many tests have special test programs that test individual functions or
```

Comment:
This paragraph is a little confusing to me. Maybe it could be reworded?

Reply:
Yes looks good, thanks. Just one typo: 
These special test programs are structured to ~have~ perform multiple individual tests

Reply:
I approve